### PR TITLE
Moved from appveyor to GH actions flow

### PR DESCRIPTION
**Pull request by Pierta**

### CODE REVIEW

**Comment by Pierta** [comment]
@lbargaoanu, @jbogard, could you, pls, take a look?

File: Build.ps1 [hunk 1]
```diff
@@ -22,21 +22,13 @@ function Exec
...
-$versionSuffix = @{ $true = "--version-suffix=$($suffix)"; $false = ""}[$suffix -ne ""]
-
-exec { & dotnet build AutoMapper.Data.sln -c Release --version-suffix=$buildSuffix -v q /nologo }
-
-exec { & dotnet test -c Release --no-build }
-
-exec { & dotnet pack .\AutoMapper.Data -c Release -o artifacts --include-symbols --no-build $versionSuffix }
-
+exec { & dotnet build AutoMapper.Data.sln -c Release -v q /nologo }
+exec { & dotnet test -c Release --no-build -l trx --verbosity=normal }
```
Comment (lbargaoanu) [inline Build.ps1:32]:
```suggestion
exec { & dotnet test -c Release -r $artifacts --no-build -l trx --verbosity=normal }
```

**Comment by lbargaoanu** [comment]
@jbogard I still see the AppVeyor check, so maybe smth else needs to be done? Maybe in the settings of the GiHub project.

**Comment by lbargaoanu** [comment]
@Pierta Thanks! https://www.nuget.org/packages/AutoMapper.Data/

**Pull request merged**